src/boosting/gbdt.cpp

```diff
@@ -547,15 +547,18 @@ void GBDT::SaveModelToFile(int num_iteration, const char* filename) const {
     output_file << pairs[i].second << "=" << std::to_string(pairs[i].first) << std::endl;
   }
 
-  output_file << std::endl << "feature information:" << std::endl;
-  for (size_t i = 0; i < max_feature_idx_ + 1; ++i) {
-    output_file << feature_names_[i] << "=" << feature_infos_[i] << std::endl;
+  if (feature_infos_.size() == feature_names_.size()) {
+      output_file << std::endl << "feature information:" << std::endl;
+      for (size_t i = 0; i < max_feature_idx_ + 1; ++i) {
+        output_file << feature_names_[i] << "=" << feature_infos_[i] << std::endl;
+      }
```

Comment:
oh...Don't fix like this way. 
The reason is I forget to load feature_infos_ in LoadModelFromString. 
Can you help to add this?